### PR TITLE
fix: empty coninuation token means last page

### PR DESCRIPTION
**Pull request by adriantam**
## Description
Fix the infinite loop problem when continuation token is empty

## References
follow up of https://github.com/openfga/cli/pull/29 

## Review Checklist
- [X] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [X] The correct base branch is being used, if not `main`
- [X] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: cmd/stores/create_test.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
+package stores
```
Comment (rhamzeh) [inline cmd/stores/create_test.go:1]: These were a different PR and unintentionally committed?

Comment (adriantam) [reply cmd/stores/create_test.go:1]: no

Comment (adriantam) [reply cmd/stores/create_test.go]: Split up the PR into two PRs (this one and one for UT only)

File: cmd/tuples/read.go [hunk 2]
```diff
@@ -19,80 +19,88 @@ import (
...
-	"os"
 
 	cmdutils "github.com/openfga/cli/lib/cmd-utils"
 	openfga "github.com/openfga/go-sdk"
 	"github.com/openfga/go-sdk/client"
 	"github.com/spf13/cobra"
 )
 
-// MaxReadPagesLength Limit the tuples so that we are not paginating indefinitely.
-var MaxReadPagesLength = 20
```
Comment (rhamzeh) [inline cmd/tuples/read.go:31]: Can you keep this? Update the init so we are referencing this instead of `MaxReadChangesPagesLength`. That was a typo.

**Pull request merged**